.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please start by describing concisely the nature of the bug or the feature
+request.
+
+In case of a bug report, please describe the steps how the problem may be
+reproduced. Does the problem occur on production servers? Does it work on the
```

Comment:
If we expect non-technical people to submit issues (and I hope they do!), it would be good to limit the use of developer-speak. :) Can we say, "Does the problem occur on (1) the CERN Open Data portal or (2) if you are a developer, a local build?" Or something to that effect?